File: bubble/bubble.js

let bubble = (arr) => {
  let i = 0
  const len = arr.length
  for (; i < len; i++) {
    let j = 1
    for (; j < len - i ; j++ ){
      if(arr[j-1] > arr[j]){
        let temp = arr[j-1]
        arr[j-1] = arr[j]
        arr[j] = temp
      }
    }
  }
  return arr
} 

let a = bubble([1111,1,6,3,4,7,232,323,12,67,89,3453,5465,12,34,5,8])
console.log(a)

File: bubble/bubble.py
def bubble(arr):
  l = len(arr)
  for i in range(l):
    for n in range(0,l-i-1):
      if arr[n] > arr[n+1]:
        arr[n+1], arr[n] = arr[n], arr[n+1]
  return arr


a = [2,45,56,575,3,565,1,564,787,111,2342,675,134]
print bubble(a)
